test/Libraries/TestServices/TestSessionConfiguration.cs

```diff
@@ -64,7 +64,7 @@ public TestSessionConfiguration(string dynamoCoreDirectory, string configFileDir
 
             LibraryVersion version;
             RequestedLibraryVersion = LibraryVersion.TryParse(versionStr, out version) ? 
-                version : LibraryVersion.Version220;
+                version : LibraryVersion.Version219;
```

Comment:
Nope. Not a mistake. This was the reversion of the original commits to remove libg219 support. That was applied on master, and also on this branch. Then we reverted it on this branch which makes it seem like I'm making this change again. When we ultimately merge this back to master, I'll get a merge conflict and have to resolve it then.